braindecode/regressor.py

```diff
@@ -250,3 +252,24 @@ def predict_proba(self, X):
             return y_pred.mean(-1)
         else:
             return y_pred
+
+    def predict_trials(self, X, return_targets=True):
+        """Create trialwise predictions (n_trials x n_classes x n_predictions),
+        and optionally also return trialwise labels (n_trials x n_targets).
+
+        Parameters
+        ----------
+        X: braindecode.datasets.BaseConcatDataset
+            A braindecode dataset to be predicted.
+        return_targets: bool
+            If True, additionally returns the trial targets.
+
+        Returns
+        -------
+        trial_predictions, trial_labels: tuple(np.ndarray, np.ndarray)
```

Comment:
please specify the dimension of each array. This syntax `tuple(np.ndarray, np.ndarray)` is odd to me.

Reply:
Dimension are at the top of the docstring https://github.com/braindecode/braindecode/pull/312/files/0c3e6db742d0012eac23e35ae1ba47f74dda7c88#diff-049a3b69cc29c2439db003260fe1ca5f9908afd5f4de2498c5ff7d155cf96f60R257-R258. Should I move it to the Returns section or add it here again?

Reply:
yes. I won't expect dimensions in the header line of a docstring. It should be in the Returns statement.

you should skim through https://www.python.org/dev/peps/pep-0257/

we should activate https://github.com/PyCQA/pydocstyle on the repo....

Reply:
I see, thank you. I support the idea for more automated checks to ensure we follow conventions.